hosted/eks/helper/helper_cluster.go

```diff
@@ -515,6 +515,164 @@ func AddNodeGroupOnAWS(nodeName, clusterName, region string, extraArgs ...string
 
 }
 
+// ScaleNodeGroupOnAWS scales nodegroup of a cluster using EKS CLI
+func ScaleNodeGroupOnAWS(ngName, clusterName, region string, numOfNodes, maxCount, minCount int64, extraArgs ...string) error {
+	fmt.Println("Scaling nodegroup of EKS cluster ...")
+	args := []string{"scale", "nodegroup", "--region", region, "--name", ngName, "--cluster", clusterName, "--nodes", fmt.Sprintf("%d", numOfNodes), "--nodes-max", fmt.Sprintf("%d", maxCount), "--nodes-min", fmt.Sprintf("%d", minCount), "--wait"}
+	if len(extraArgs) != 0 {
+		args = append(args, extraArgs...)
+	}
+	fmt.Printf("Running command: eksctl %v\n", args)
+	out, err := proc.RunW("eksctl", args...)
+	if err != nil {
+		return errors.Wrap(err, "Failed to scale nodegroup: "+out)
+	}
+	fmt.Println("Scaled nodegroup: ", ngName)
+	return nil
+}
+
+// UpdateNodeGroupLabelsOnAWS deletes or add/updates labels on nodegroup of a cluster;
+func UpdateNodeGroupLabelsOnAWS(clusterName, nodegroupName, region string, addOrUpdatelabels map[string]string, removeLabels []string, extraArgs ...string) error {
```

Comment:
>There are no specific commands in eksctl to update the labels of a nodegroup, but it can easily be achieved using kubectl, e.g.:

Ref: https://eksctl.io/usage/nodegroups/#update-labels